File: backend/components/metiers/threed/security/docs/samples/audit/samples/sample_audit.py
# sample_audit.py
# Exemple d’intégration avancée pour l’audit

from helpers_audit import log_event, alert_on_suspicious_activity

log_event('LOGIN', {'user': 'Alice', 'status': 'success'})
alert = alert_on_suspicious_activity({'level': 'high', 'action': 'delete'})
print(alert)

File: backend/components/metiers/threed/security/docs/samples/audit/samples/sample_audit.js
// sample_audit.js
// Exemple d’intégration avancée pour l’audit

const { logEvent, alertOnSuspiciousActivity } = require('./helpers_audit');

logEvent('LOGIN', { user: 'Alice', status: 'success' });
const alert = alertOnSuspiciousActivity({ level: 'high', action: 'delete' });
console.log(alert);
